src/checkers/inference/solver/frontend/TwoQualifiersLattice.java

```diff
@@ -14,41 +14,13 @@
  */
 public class TwoQualifiersLattice extends Lattice {
```

Comment:
Do we need to know if it is a TwoQualifiersLattice from outside? If not, maybe this class can be removed as it is the same as super class, except it's class name.